uniqueidentifier_nullable.go

```diff
@@ -0,0 +1,37 @@
+package mssql
```

Comment:
I purposefully named this `uniqueidentifier_nullable.go` and not `nullable_uniqueidentifier.go` to keep these file next to each other. Any better ideas?

Reply:
seems reasonable to me